sensu/plugins/metrics-ceph.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
```

Comment:
Future proofing, mostly.  I don't use its features here.  This is 2.6+ compatible so should be no issue.